tests/integrations/storage/test_storage.rs

```diff
@@ -1093,16 +1094,27 @@ fn test_txn_store_rawkv_api_version() {
                     vec![(key.to_vec(), b"value".to_vec())],
                 );
 
-                // TODO: Remove this conditional statement after raw_checksum is done for API V2
-                if storage_api_version != ApiVersion::V2 {
-                    let mut digest = crc64fast::Digest::new();
-                    digest.write(key);
-                    digest.write(b"value");
-                    store.raw_checksum_ok(
-                        vec![range_bounded.clone()],
-                        (digest.sum64(), 1, (key.len() + b"value".len()) as u64),
-                    );
-                }
+                let digest = crc64fast::Digest::new();
+                let ttl = if storage_api_version == ApiVersion::V1 {
+                    Some(0)
+                } else {
+                    None
+                };
+                let (engine_key, engine_value) = dispatch_api_version!(storage_api_version, {
```

Comment:
`encoded_key` & `encoded_value` ? We never use `engine_key` & `engine_value` before.

Reply:
done.